src/client/java/teammates/client/scripts/util/Stopwatch.java

```diff
@@ -0,0 +1,28 @@
+package teammates.client.scripts.util;
+
+import teammates.common.util.Logger;
+
+/**
+ * Helper class to measure event durations.
+ */
+public class Stopwatch {
+    private long startTime;
+
+    /**
+     * Starts the Stopwatch.
+     */
+    public void start() {
+        startTime = System.nanoTime();
+    }
+
+    private double getTimeElapsedInSeconds() {
+        return (System.nanoTime() - startTime) / 1000000000.0;
+    }
+
+    /**
+     * Logs the time elapsed since the start using the supplied logger.
+     */
+    public void logTimeElapsedInSeconds(Logger log) {
+        log.info("Time taken: " + getTimeElapsedInSeconds());
+    }
```

Comment:
As mentioned earlier, using print statements quite often result in the output appearing out of order with the test descriptions that appear in the console.